lib/origin_simulator/payload.ex

```diff
@@ -21,11 +21,12 @@ defmodule OriginSimulator.Payload do
     GenServer.call(server, {:generate, value, route})
   end
 
-  def body(_server, status, route \\ Recipe.default_route()) do
-    case status do
-      200 -> cache_lookup(route)
-      404 -> {:ok, "Not found"}
-      406 -> {:ok, "Recipe not set, please POST a recipe to /add_recipe"}
+  def body(_server, status, path \\ Recipe.default_route(), route \\ Recipe.default_route()) do
+    case {status, path} do
+      {200, _} -> cache_lookup(route)
+      {404, _} -> {:ok, "Not found"}
+      {406, "/*"} -> {:ok, OriginSimulator.recipe_not_set()}
+      {406, _} -> {:ok, OriginSimulator.recipe_not_set(path)}
```

Comment:
are `{406, "/*"}` and `{406, _}` not the same? :)

Reply:
For backward compatibility, the message is different for "/*", which now a default catch-all route. Its usage is implicit for the previous use cases, e.g. "any route" simulation (for recipes without routes). The "recipe not set" message for "/*" is generic and has no mention of route.